packages/shared/src/store/dbHooks.ts

```diff
@@ -200,7 +200,7 @@ export const useContacts = () => {
 
 export const useUnreadsCount = () => {
   return useQuery({
-    queryKey: ['unreadsCount'],
+    queryKey: ['unreadsCount', useKeyFromQueryDeps(db.getUnreadsCount)],
```

Comment:
this works but causes a lot of seemingly irrelevant query updates – would love to know if there's a way to reduce

Reply:
Ya, I mean this is true in a number of places, mostly because our query invalidation is very coarse. I think fine for now + we should address at a higher level in a perf pass sometime.